config/rbac/submariner-diagnose/role.yaml

```diff
@@ -33,14 +33,19 @@ rules:
   - apiGroups:
       - submariner.io
     resources:
-      - '*'
+      - clusters
```

Comment:
Does it need globalnet CR( like ClusterGlobalEgressIP) permission too ?

Reply:
Those are cluster-scoped so specified in the ClusterRole.